client/src/pages/Profile.js

```diff
@@ -70,7 +70,7 @@ const Profile = () => {
       .catch((err) => {
         logout();
       });
-  }, []);
```

Comment:
The `, []` was to make sure the useEffect only ran once; are you sure it is okay to remove this?